server/routes/updateGoal/mappers/goalToUpdateGoalFormMapper.test.ts

```diff
@@ -9,13 +9,13 @@ describe('goalToUpdateGoalFormMapper', () => {
       const goalReference = '1a2eae63-8102-4155-97cb-43d8fb739caf'
       const step1 = aValidStep()
       const step2 = anotherValidStep()
-      const goal = aValidGoal(goalReference, [step1, step2])
+      const goal = aValidGoal({ goalReference, steps: [step1, step2] })
```

Comment:
I changed `aValidGoal` test data builder to take an options object rather than positional arguments
(like we are doing with all our other test data builders)